src/Collections/Sites.php

```diff
@@ -190,41 +179,74 @@ public function findUuidByName($name)
     }
 
     /**
+     * Get the site's UUID given either it's site name or it's UUID
+     *
+     * @param $id
+     * @return string
+     */
+    public function findUuid($id)
```

Comment:
Ok, I've made a change to this to break it up a bit, make the functions return what they say they return, and make the whole thing protected. That makes the lookup command a bit less efficient but it gives us less surface area on the API to worry about so I think that's a good tradeoff. These functions are still a bit clunky but since they're now internal to the collection and not part of an API we need to support forever I think it's good enough for now.